config/seatunnel.yaml

```diff
@@ -37,3 +37,8 @@ seatunnel:
     telemetry:
       metric:
         enabled: false
+    http:
+      enable-http: true
+      port: 8080
+      context-path: /seatunnel
```

Comment:
I wonder if we should not provide `context-path` configuration, because this will lead to inconsistent URL addresses and there is no benefit.

Reply:
> I wonder if we should not provide `context-path` configuration, because this will lead to inconsistent URL addresses and there is no benefit.

Configuring context-path helps unify and organize URLs, enhances deployment flexibility, supports multiple applications and microservices architecture, and improves security and maintenance. Removing this configuration can lead to management chaos and inconvenience.